bin/ccd-roles.json

```diff
@@ -28,5 +28,6 @@
   { "role": "manage-translations", "security_classification": "PUBLIC" },
   { "role": "load-translations", "security_classification": "PUBLIC" },
   { "role": "citizen", "security_classification": "PUBLIC" },
-  { "role": "pui-caa", "security_classification": "PUBLIC" }
+  { "role": "pui-caa", "security_classification": "PUBLIC" },
+  { "role": "next-hearing-date-admin", "security_classification": "PUBLIC" }
```

Comment:
next-hearing-date-updater-admin ??

Reply:
I could change it but this would lead to change the original HMAN-320/326.  This is user/role  defined already.

Reply:
This is the value (`next-hearing-date-admin`) specified in the [Scope of Work](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1588729072#MaintainNextHearingDateScopeofWork-Authentication)

Reply:
Cheers @sullivangareth